src/templates/blog/post.js

```diff
@@ -40,9 +42,23 @@ export const query = graphql`
         }
       }
     }
+    seoImageFile: file(absolutePath: { eq: $seoImage }) {
+      childImageSharp {
+        fixed(width: 2160, height: 1080) {
+          ...GatsbyImageSharpFixed_noBase64
+        }
+      }
+    }
   }
 `
 
+const resolveImageURL = ({ file, src }) =>
+  pathToURL(_get(file, "childImageSharp.fixed.src") || src)
```

Comment:
_get 3rd argument is the default value, you can use that.

Reply:
True story, my bad :+1: